finetune/base_models/bert/featurizer.py

```diff
@@ -148,3 +161,695 @@ def get_zeros():
 
 
 layoutlm_featurizer = functools.partial(bert_featurizer, underlying_model=LayoutLMModel)
+xdoc_featurizer = functools.partial(bert_featurizer, underlying_model=XDocModel)
```

Comment:
Move all of this to it's own file. (will have to wait until current run of experiments are complete)